content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "CockroachDB"
+metaTitle: "CockroachDB"
+metaDescription: "Learn how to use CockroachDB with Palette Dev Engine"
+hideToC: false
+type: "appTier"
+category: ['databases']
+hiddenFromNav: false
+fullWidth: false
+logoUrl: "https://upload.wikimedia.org/wikipedia/en/thumb/3/31/Cockroach_Labs_Logo.png/220px-Cockroach_Labs_Logo.png"
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# CockroachDB
+
+CockroachDB is a [distributed SQL database](https://www.cockroachlabs.com/blog/what-is-distributed-sql/) designed for cloud-native environments, providing a reliable and scalable solution for managing data across multiple nodes and regions. Its architecture automates data replication, sharding, and rebalancing, simplifying operational tasks and enabling developers to concentrate on building their applications.
```

Comment:
```suggestion
CockroachDB is a [distributed SQL database](https://www.cockroachlabs.com/blog/what-is-distributed-sql/) designed for cloud-native environments. CockroachDB provides a reliable and scalable solution for managing data across multiple nodes and regions. Its architecture automates data replication, sharding, and rebalancing, By simplifying operational tasks, Cockroach enables developers to concentrate on building their applications.
```

Reply:
It's up to you if you want to use the above suggestion. I offer it only to break up some long sentences.